### PR TITLE
drakcore: add drak-install-ipt-tools command

### PR DESCRIPTION
**Pull request by icedevml**
Automatically install the required custom IPT tools.

### CODE REVIEW

File: drakcore/setup.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
              'drakcore/bin/drak-healthcheck',
              'drakcore/bin/drak-vncpasswd',
              'drakcore/bin/drak-upgrade-db',
+             'drakcore/bin/drak-install-ipt-tools',
```
Comment (BonusPlay) [inline drakcore/setup.py:23]: indent?

File: drakcore/drakcore/bin/drak-install-ipt-tools [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+mkdir /tmp/iptbuild
+cd /tmp/iptbuild
+
+git clone https://github.com/icedevml/libipt.git
+git clone https://github.com/intelxed/xed.git
+git clone https://github.com/intelxed/mbuild.git
+
+cd xed
+./mfile.py --share
+sudo ./mfile.py --prefix=/usr/local install
```
Comment (BonusPlay) [inline drakcore/drakcore/bin/drak-install-ipt-tools:14]: Maybe check if this is running as root at the start?

**Review by BonusPlay — CHANGES_REQUESTED** [review]
I don't like this approach. It adds random script that has dependencies on multiple packages (building tools) which are not required by drakvuf-sandbox atm. Imo this should be either another deb package or it should stay in documentation.

File: drakcore/setup.py (hunk 1 above)
Comment (icedevml) [reply drakcore/setup.py:23]: :thinking: what's wrong with this?

**Pull request closed without merging**